package.json

```diff
@@ -30,14 +30,12 @@
     "node": ">=14.0.0"
   },
   "devDependencies": {
-    "@babel/core": "^7.11.6",
-    "@babel/plugin-proposal-class-properties": "^7.10.4",
-    "@babel/plugin-proposal-export-default-from": "^7.10.4",
```

Comment:
These two plugins are part of the language now and don't have to be installed manually